platform/device-manager.ts

```diff
@@ -3,24 +3,18 @@
  * TODO(cjdaly) integrate with Mock Local Home Platform and Mock Network
```

Comment:
should we review the common trunk of changes as part of #29 or #28?

It may help to rebase the change on top of each others branches (rather than having both against master) if they are truely dependent on each other.

Reply:
Yeah they are dependent on each other I will look into this.

Reply:
I'm assuming I won't be able to open a PR on this repo with one branch targeting another branch if both of the branches exist on a fork (haven't tested this).  I added a reference to #29 to the description of this PR.  For now I think we should review the common changes on this PR.  I'll continue looking into the best way to do this though.

Reply:
For what it's worth, I can confirm I can't create a PR on this repo with both branches on my fork.